coalib/bears/Bear.py

```diff
@@ -12,8 +12,10 @@
 from coala_utils.decorators import (enforce_signature, classproperty,
                                     get_public_members)
 
+# Ignore PyUnusedCodeBear
 from coalib.bears.requirements.PackageRequirement import PackageRequirement
-from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.bears.requirements.PipRequirement import PipRequirement \
+                                                    # Ignore PyUnusedCodeBear
```

Comment:
almost all the ignore comments were added by coala itself, except the ones which conflict with the LineLenghtBear.